test/bitarray.jl

```diff
@@ -73,6 +73,10 @@ allsizes = [((), BitArray{0}), ((v1,), BitVector),
     @test !any(d)
     @test all(b)
     @test sz == size(d)
+    for ii in 1:length(sz)
+        @test isassigned(a, ii)
+        @test isassigned(b, ii)
+    end
```

Comment:
Could be nice to also test `!isassigned` for some out of bounds `ii` while you're at it.